incubator/virtualcluster/cmd/vn-agent/app/server.go

```diff
@@ -136,5 +137,12 @@ func Run(c *config.Config, serverOption *options.ServerOption, stopCh <-chan str
 		klog.Errorf("server listen error %v", err)
 	}
 
+	go func() {
+		// start a health http server.
+		mux := http.NewServeMux()
+		healthz.InstallHandler(mux)
+		klog.Fatal(http.ListenAndServe(":8080", mux))
```

Comment:
I think vn-agent will use host network. The 8080 port may not be available.

Reply:
changed port to 8088

Reply:
it is unnecessary to open a health port for vn-agent. firstly it occupy a node port. node port should be strictly managed for security consideration.  And it may conflict with other component.
Using tcpSocket type liveness from kubelet is enough for vn-agent.